masonry/src/theme.rs

```diff
@@ -79,6 +79,9 @@ static DEBUG_COLOR: &[Color] = &[
     Color::rgb8(0, 0, 0),
 ];
 
+/// A color used for debug painting.
+///
+/// The same color is always returned given the same id, usually the id of a widget.
 pub fn get_debug_color(id: u64) -> Color {
```

Comment:
Can we add a method to `PaintCtx` for this? As a standalone function, this is a bit undiscoverable.

Keeping this as a raw method is still correct, but since the vast majority of potential users will be calling it with `ctx.widget_id().to_raw()`, we can improve ergonomics and discoverability.